packages/core-ethereum/src/ethereum.ts

```diff
@@ -193,9 +174,48 @@ export async function createChainWrapper(
       throw new Error(`Failed in mining transaction. ${error}`)
```

Comment:
The error message is misleading, we haven't mined to tx yet.

Reply:
changed